src/main/resources/swagger/api-docs.yaml

```diff
@@ -4717,4 +4718,7 @@ components:
     oidc:
       type: openIdConnect
       openIdConnectUrl: OPEN_ID_CONNECT_URL
-      x-tokenName: id_token
\ No newline at end of file
+      x-tokenName: id_token
+    bearerAuth:
```

Comment:
do you need `bearerAuth` given you are adding oidc as a `HttpBearerAuth`?

Reply:
`bearerAuth` adds `ApiClient.setBearerToken()`

Without it I can still do:
```
    ((HttpBearerAuth) apiClient.getAuthentications().get("oidc")).setBearerToken(token);
```
but that feels a little unwieldy.

Reply:
I guess another idea is I could mustache `setAccessToken` to set the bearer token for `oidc`. But I kinda want to limit how much custom code we add…

Reply:
why doesn't `apiClient.setBearerToken(token);` work?

Reply:
We connected in slack. `apiClient.setBearerToken` is conditionally only added when `bearerAuth` is included in the YAML.

We think it's reasonable to support both oidc (for interactive flows) and bearer (for programmatic access) security schemes.